sdk/docs/pages/typescript/cryptography/keypairs.mdx

```diff
@@ -126,6 +126,22 @@ const exampleMnemonic = 'result crisp session latin ...';
 const keyPair = Ed25519Keypair.deriveKeypair(exampleMnemonic);
 ```
 
+## Deriving a `Keypair` from a Bech32 encoded secret key
+
+You can derive a `Keypair` by converting the 33-byte secret key encoded in Bech32 string to a 
+`Uint8Array` and passing it to the `fromSecretKey` method of a `Keypair` class.
+
+```typescript
+import { Ed25519Keypair } from '@mysten/sui.js/keypairs/ed25519';
+import { decodeSuiPrivateKey, encodeSuiPrivateKey } from '@mysten/sui.js/cryptography/keypair';
+
+const encoded = encodeSuiPrivateKey([59, 148, 11, 85, 134, 130, 61, 253, 2, 174, 59, 70, 27, 180, 51, 107, 94, 203, 174, 253, 102,39, 170, 146, 46, 252, 4, 143, 236, 12, 136, 28], 'ED25519');
```

Comment:
Might be a stupid question, but if the Bech32 secret key is 33 bytes, and the `encodeSuiPrivateKey()` accepts 32 bytes, which byte should we omit?

Reply:
see in the function documentation: encodePrivateKey takes a 32-byte array and a signature scheme

```
/**
 * This returns a Bech32 encoded string starting with `suiprivkey`,
 * encoding 33-byte `flag || bytes` for the given the 32-byte private
 * key and its signature scheme.
 */
export function encodeSuiPrivateKey(bytes: Uint8Array, scheme: SignatureScheme): string {
	if (bytes.length !== PRIVATE_KEY_SIZE) {
		throw new Error('Invalid bytes length');
	}
	const flag = SIGNATURE_SCHEME_TO_FLAG[scheme];
	const privKeyBytes = new Uint8Array(bytes.length + 1);
	privKeyBytes.set([flag]);
	privKeyBytes.set(bytes, 1);
	return bech32.encode(SUI_PRIVATE_KEY_PREFIX, bech32.toWords(privKeyBytes));
}
```

Reply:
Thanks!